src/new-client/src/App/Trades/TradesGrid/TableHeadCell.tsx

```diff
@@ -34,42 +43,104 @@ const TableHeadCell = styled.th`
     display: inline-block;
   }
 `
+const AlignedFilterIcon = styled(FaFilter)`
+  margin-left: 0.2rem;
+  margin-right: 0.1rem;
+`
+
+export const ContactUsPopup = styled(Popup)`
+  box-shadow: 0 7px 26px 0 rgba(23, 24, 25, 0.5);
+  bottom: calc(2rem + 0.25rem);
+  border-radius: 0.5rem;
+`
 
 export const TableHeadCellContainer: React.FC<{
   field: ColField
 }> = ({ field }) => {
   const [showFilter, setShowFilter] = useState(false)
-  const ref = useRef<HTMLDivElement>(null)
+  const ref = useRef<HTMLTableHeaderCellElement>(null)
   const { displayMenu, setDisplayMenu } = usePopUpMenu(ref)
   const tableSort = useTableSort()
   const { headerName, filterType } = colConfigs[field]
+  const numeric = filterType === "number" && field !== "tradeId"
```

Comment:
If trade ids have natural arithmetic order (which in our case they do), you might want to filter by aspects of that order.  If we don't want to rely on that (maybe the nature of an idea will change the future), in maybe we make "filterType" optional and there are just some columns we don't allow filtering on.